docs/list_of_libaries.md

```diff
@@ -9,11 +9,40 @@
 * - Title
   - Link
   - Description
-* - Ingest service
-  - [https://github.com/sensein/BrainKB/tree/ingestion-fapi-skeleton/ingestion_service](https://github.com/sensein/BrainKB/tree/ingestion-fapi-skeleton/ingestion_service)
-  - 
+* - BrainKB
+  - [sensein/BrainKB](https://github.com/sensein/BrainKB)
+  - Source code for different microservice components of BrainKB.
+* - BrainKB UI
+  - [sensein/brainkb-ui](https://github.com/sensein/brainkb-ui)
+  - Source code for the BrainKB user interface ([https://beta.brainkb.org](https://beta.brainkb.org)).
+* - ProvSense
+  - [pypi.org/project/ProvSense](https://pypi.org/project/ProvSense)
+  - A Python library for comparing changes in KGs.
+* - GrobidArticleExtractor
+  - [pypi.org/project/GrobidArticleExtractor](https://pypi.org/project/GrobidArticleExtractor)
+  - A Python library designed to extract and organize content from scientific papers in PDF format.
+* - EviSense
+  - [pypi.org/project/EviSense](https://pypi.org/project/EviSense)
+  - A Python library to extract evidence and rationales for specific terms within documents, including scientific publications. It supports multiple LLM providers (e.g., Ollama and OpenRouter) and allows the use of multiple models for greater flexibility. See the [example](https://github.com/sensein/EviSense/tree/experiment/example) for a demonstration of how to use it.  
+* - SchemaExtractor
+  - [sensein/SchemaExtractor](https://github.com/sensein/SchemaExtractor/tree/dev)
+  - A Python library for extracting and analyzing schemas from knowledge graphs, leveraging llm-based multi-agent autonomous systems for efficient processing and analysis.
 ```
 
+## Ontologies/Schemas
+```{list-table} Ontologies/Schemas Developed as Part of BrainKB
```

Comment:
Perhaps the subheading is a bit redundant?

![image](https://github.com/user-attachments/assets/c9cf1143-4af3-4419-8aa0-da454209c696)

```suggestion
```{list-table}
```